lib/util.js

```diff
@@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
 function isBuffer(arg) {
   return arg instanceof Buffer;
 }
-exports.isBuffer = isBuffer;
+exports.isBuffer = deprecate(isBuffer,
+    'util.isBuffer is deprecated, please use a user-land alternative.');
```

Comment:
Ooops. Honestly, I don't even know why we use `Buffer.isBuffer` either... it's just `instannceof Buffer`...